Makefile

```diff
@@ -57,11 +57,11 @@ endif
 .PHONY: escapes
 escapes: ## golang memeory escapes check
 ifeq ($(CLOUD_PROVIDER),libvirt)
-	go build $(GOFLAGS) -gcflags="-m -l" -o cloud-api-adaptor cmd/cloud-api-adaptor/main.go | grep "escapes to heap" || true
-	go build $(GOFLAGS) -gcflags="-m -l" -o agent-protocol-forwarder cmd/agent-protocol-forwarder/main.go | grep "escapes to heap" || true
+	go build $(GOFLAGS) -gcflags="-m -l" -o cloud-api-adaptor cmd/cloud-api-adaptor/main.go 2>&1 | grep "escapes to heap" || true
```

Comment:
for the problem here in fact it's related to/limited by our project structure.
```
yuanyi@yuanyideMacBook-Pro cloud-api-adaptor % go build -gcflags="-m -l" -o cloud-api-adaptor ./cmd/cloud-api-adaptor ./pkg
directory cmd/cloud-api-adaptor is contained in a module that is not one of the workspace modules listed in go.work. You can add the module to the workspace using:
        go work use .
no Go files in /Users/yuanyi/go/src/github.com/SamYuan1990/cloud-api-adaptor/pkg
```
Let's find some way to make it better. as for now, we have latest golang as 1.20 for now. (doesn't mean we need to bump up version to 1.20 for now, just figure out solution.)

Reply:
if we make it just to package/folder, there will be nothing detected. for example: https://github.com/confidential-containers/cloud-api-adaptor/actions/runs/4163646155/jobs/7204298902#step:6:8

Reply:
Thank you for finding out how to check all files other than `main.go`

I post some comments at https://github.com/confidential-containers/cloud-api-adaptor/issues/604